internal/report/humanprinter_test.go

```diff
@@ -16,7 +17,8 @@ func TestUserFriendlyPrinter_Print(t *testing.T) {
 		name            string
 		vulnerabilities []vulnerability
 		sum             summary
-		wantNilErr      bool
+		wantSum         []string
+		wantVuln        []string
```

Comment:
I would mix both in a single field:

```suggestion
		want         []string
```